ha/monitor/k8s/parser.py

```diff
@@ -58,11 +61,13 @@ def _create_health_alert(self, res_type: str, res_name: str, health_status: str)
         res_name: actual resource name. Ex: machine_id in case of node alerts
         health_status: health of that resource. Ex: online, failed
         """
-        self.event = HealthEvent()
         self.payload[HealthAttr.RESOURCE_TYPE.value] = res_type
         self.payload[HealthAttr.RESOURCE_ID.value] = self.payload[HealthAttr.NODE_ID.value] = res_name
         self.payload[HealthAttr.RESOURCE_STATUS.value] = health_status
-        self.event.set_payload(self.payload)
+
+        # Create event schema object with payload data also
+        # can change value of payload attributes with set function
+        self.event = HealthEvent(**self.payload)
```

Comment:
why not called KvPayload(self.payload) or is it getting called internally.
I have approved it is just a query.